src/main/scala/mesosphere/marathon/core/matcher/base/util/ActorOfferMatcher.scala

```diff
@@ -27,6 +28,14 @@ class ActorOfferMatcher(actorRef: ActorRef, override val precedenceFor: Option[P
   }
 
   override def toString: String = s"ActorOfferMatcher($actorRef)"
+
+  override def isInterestedIn(offer: Offer) = {
+    def isFromHomeRegion(offer: Offer): Boolean = {
```

Comment:
Out of curiosity, why is this inside the method and not outside?

Reply:
I introduced this function just to improve readability so I like to nest it within the function that's using it (if it is actually not that meaningful for ActorOfferMatcher itself)